src/test/java/org/broadinstitute/hellbender/utils/codecs/GENCODE/GencodeGtfCodecUnitTest.java

```diff
@@ -0,0 +1,3336 @@
+                null, "SELENOO", GencodeGtfFeature.GeneTranscriptType.protein_coding, null, "SELENOO-001", 5, "ENSE00003503715.1", GencodeGtfFeature.LocusLevel.TWO,
+                new ArrayList<>(
+                        Arrays.asList(
+                                new GencodeGtfFeature.OptionalField<>("protein_id", "ENSP00000370288.2"),
+                                new GencodeGtfFeature.OptionalField<>("transcript_support_level", GencodeGtfFeature.TranscriptSupportLevel.ONE),
+                                new GencodeGtfFeature.OptionalField<>("tag", GencodeGtfFeature.FeatureTag.basic),
+                                new GencodeGtfFeature.OptionalField<>("tag", GencodeGtfFeature.FeatureTag.appris_principal_2),
+                                new GencodeGtfFeature.OptionalField<>("tag", GencodeGtfFeature.FeatureTag.CCDS),
+                                new GencodeGtfFeature.OptionalField<>("tag", GencodeGtfFeature.FeatureTag.seleno),
+                                new GencodeGtfFeature.OptionalField<>("ccdsid", "CCDS43034.1"),
+                                new GencodeGtfFeature.OptionalField<>("havana_gene", "OTTHUMG00000044645.3"),
+                                new GencodeGtfFeature.OptionalField<>("havana_transcript", "OTTHUMT00000075003.2")
+                        )
+                ),
+                null
+        );
+
+        GencodeGtfExonFeature exon16 = (GencodeGtfExonFeature) GencodeGtfFeature.create(45, "chr22", GencodeGtfFeature.AnnotationSource.HAVANA, GencodeGtfFeature.FeatureType.exon,
+                50215717, 50215867, GencodeGtfFeature.GenomicStrand.FORWARD, GencodeGtfFeature.GenomicPhase.DOT, "ENSG00000073169.13", "ENST00000380903.6", GencodeGtfFeature.GeneTranscriptType.protein_coding,
+                null, "SELENOO", GencodeGtfFeature.GeneTranscriptType.protein_coding, null, "SELENOO-001", 6, "ENSE00003573348.1", GencodeGtfFeature.LocusLevel.TWO,
+                new ArrayList<>(
+                        Arrays.asList(
+                                new GencodeGtfFeature.OptionalField<>("protein_id", "ENSP00000370288.2"),
+                                new GencodeGtfFeature.OptionalField<>("transcript_support_level", GencodeGtfFeature.TranscriptSupportLevel.ONE),
+                                new GencodeGtfFeature.OptionalField<>("tag", GencodeGtfFeature.FeatureTag.basic),
+                                new GencodeGtfFeature.OptionalField<>("tag", GencodeGtfFeature.FeatureTag.appris_principal_2),
+                                new GencodeGtfFeature.OptionalField<>("tag", GencodeGtfFeature.FeatureTag.CCDS),
+                                new GencodeGtfFeature.OptionalField<>("tag", GencodeGtfFeature.FeatureTag.seleno),
+                                new GencodeGtfFeature.OptionalField<>("ccdsid", "CCDS43034.1"),
+                                new Gencod
```

Comment:
Methods that generate expected data for a particular file should all have the correct filename in the method name, to avoid potential confusion.